test/pylib/resource_gather.py

```diff
@@ -46,7 +53,10 @@ def get_cgroup() -> Path:
     # Extract the relative cgroup for the process and make it absolute and add where the test.py process should be
     # placed in.
     # This can be used to manipulate the cgroup's controllers
-    return Path(f"/sys/fs/cgroup/{cgroup_info[0].strip().split(':')[-1]}/initial")
+    cgroup_name = Path(f'/sys/fs/cgroup/{cgroup_info[0].strip().split(":")[-1]}')
+    if cgroup_name.stem != 'resource_gather':
+        cgroup_name = cgroup_name / 'resource_gather'
+    return cgroup_name
```

Comment:
There are two changes here:
1. s/initial/resource_gather/
2. handle the case when the "resource_gather" is already there

So two questions
1. Why renaming it?
2. Wasn't it possible before this PR?